it/rcea/src/test/java/com/linecorp/armeria/common/CyclicDependencyTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common;
+
+import org.junit.jupiter.api.Test;
+
+public class CyclicDependencyTest {
```

Comment:
How about prefixing `Flags` to tell what could have cyclic dependencies.
```suggestion
class FlagsCyclicDependencyTest {
```

Reply:
Forgot about this comment 😅  Will handle renaming soon

module: `rcea` -> `flags-cyclic-dep` (since I doubt other tests will be added in this module)
test class: `CyclicDependencyTest ` -> `FlagsCyclicDependencyTest`
testn ame: `basicCase`